### PR TITLE
update nagios to notification integration

### PR DESCRIPTION
**Pull request by ronsherf**
Update from log integration to notification integration

Resolve INT-101

### CODE REVIEW

**Comment by ronsherf** [comment]
@rasienko Please have a look at the text, related to task INT-101
(Default Nagios integration should base on Notifications)
Please take a look at the next link:
 https://odron-a.bigpanda.io/#/app/integrations/nagios.production/instructions/nagios

File: _integrations/agent-common/configure-agent-actions/generic.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#### Configure the BigPanda Agent
+
+1. Configure the agent to work with Nagios Notifications integration by running the following commands:
+
+        bigpanda-config --init --token $TOKEN
+        bigpanda-config --add nagios-notification --app-key $STREAM_ID
+
+2. Add the Nagios user to the bigpanda group by running the following command:
+
+        sudo usermod -a -G bigpanda <Nagios username>
```
Comment (hkariti) [inline _integrations/agent-common/configure-agent-actions/generic.md]: is this new? didn't we use to add bigpanda to the nagios group, and not the other way around?

Comment (ronsherf) [inline _integrations/agent-common/configure-agent-actions/generic.md]: @erikzaadi

Comment (m3nadav) [inline _integrations/agent-common/configure-agent-actions/generic.md]: @hkariti In nagios notifications, nagios is the one running the bigpanda-notification command which creates a file in a bigpanda queue folder, thats how we give nagios the right permissions to do that.

**Comment by ronsherf** [comment]
@erikzaadi please review and approve

**Comment by erikzaadi** [comment]
image

**Pull request merged**